drivers/issi/is31fl3733.c

```diff
@@ -213,11 +228,15 @@ void IS31FL3733_set_led_control_register(uint8_t index, bool red, bool green, bo
 
 void IS31FL3733_update_pwm_buffers(uint8_t addr, uint8_t index) {
     if (g_pwm_buffer_update_required[index]) {
-        // Firstly we need to unlock the command register and select PG1
+        // Firstly we need to unlock the command register and select PG1.
         IS31FL3733_write_register(addr, ISSI_COMMANDREGISTER_WRITELOCK, 0xC5);
```

Comment:
you aren't using return value at all? is that ok

Reply:
yes pwm buffer is updated all the time. so even if a transaction fails (ie a frame is faulty) the next one will be alright very quickly